index.html

```diff
@@ -192,6 +192,28 @@ <h2>About Myself</h2>
             </section>
         </div>
 
+        <section class="contact-me-section">
+            <div class="contact-details">
+                <div class="contact-me">
+                    <h2>Contact me</h2>
+                </div>    
+                <p class="paragraph">
+                    If you have an application you are interested in developing,
+                    a feature that you need built or a project that needs coding.
+                    I’d love to help with it
+                </p>
+                <form class="input-field" action="https://formspree.io/f/xdobrban" method="post" >
+                    <input class="name-input" type="text" name="name" maxlength="30" required placeholder="Your name">
+                    <input class="email-input" type="email" name="email" required placeholder="Your email">
+                    <textarea class="textarea-input" name="message" maxlength="500" required cols="30" rows="10"
+                     placeholder="Write your message here"></textarea>

```

Comment:
- [ ] Kindly you should stick to the design as much as possible. try to increase the padding of form inputs to match the Figma design as much as possible so that your website can look good.

your design:
![image](https://user-images.githubusercontent.com/90524466/178589303-d1090751-0097-47e8-852b-dc5142e97b0b.png)


Figma design:
![image](https://user-images.githubusercontent.com/90524466/178588122-165f3f61-a792-47f9-981a-1ed7fc77de00.png)

Reply:
@leonardodiasb,  can you please clarify the issue padding for the input fields you have mentioned in the review. I'm not able to quite follow you in this case. Thanks